CONTRIBUTING.md

```diff
@@ -35,54 +36,103 @@ $ urbit my-fake-zod
 
 ## Git practice
 
-Since we use the GitHub issue tracker, it is helpful (though not required) to
-contribute via a GitHub pull request. If you already know what you are doing,
-skip down to the Style section.
+### Contributing
 
-Start by cloning the repository on your work machine:
+The canonical source tree is located in the `master` branch at
+[https://github.com/urbit/urbit][repo].  You should typically branch off of
+`master` when commencing new work; similarly, when we pull in your
+contribution, we'll do so by merging it to `master`.
 
-```
-$ git clone https://github.com/urbit/urbit
-```
+Since we use GitHub, it's helpful, though not required, to contribute via a
```

Comment:
```suggestion
Since we use GitHub it's helpful, though not required, to contribute via a
```

Reply:
This sentence feels pretty slow/stutter-y as-is though, imo.

Reply:
@Fang- since you're the chap who's requesting changes: how badly do you want this changed? 😄

Reply:
I just don't think the comma-heavy style makes for good docs. But I wouldn't want to die for this. So, uh, 70%?

Reply:
Fixed in 174e194.